# 客户端请求时发送
int32 num1
---
# 服务器响应发送的数据
int32 health_flag
int32 error_count
int32[] error_codes
string[] error_msgs